sparv/api/classes.py

```diff
@@ -611,20 +611,6 @@ def ungzip(self, out: str):
                 f.write(data)
         logger.info("Successfully unzipped %s", out)
 
-    def rename(self, new_name: Union[str, pathlib.Path]):
```

Comment:
I suggest adding the following line at line 592:
```
        logger.debug("Downloading from: %s", url)
```

Reply:
✅